src/__experimental__/components/decimal/decimal.spec.js

```diff
@@ -1649,6 +1619,46 @@ describe("Decimal", () => {
       expect(hiddenValue()).toBe("");
     });
 
+    it.each(["^", "ˇ", "˝", "¨", "¯", "´", "~"])(
+      "handles (%s) symbol without error",
+      (deadKey) => {
+        const onBlur = jest.fn();
+        render({ value: "0.00", onBlur });
+        setProps({ value: `${deadKey}` });
+        blur();
+        expect(onBlur).toHaveBeenCalled();
+        expect(value()).toBe("0.00");
+        expect(hiddenValue()).toBe("0.00");
+      }
+    );
+
+    it.each([
+      "ŵ",
+      "ê",
+      "û",
+      "î",
+      "ô",
+      "â",
+      "á",
+      "ã",
+      "æ",
+      "ç",
+      "ë",
+      "í",
+      "ñ",
+      "ö",
+      "õ",
+      "ø",
+    ])("handles accented letter (%s) without error", (accentedLetter) => {
+      const onBlur = jest.fn();
+      render({ value: "20.00", onBlur });
+      setProps({ value: `${accentedLetter}` });
+      blur();
+      expect(onBlur).toHaveBeenCalled();
+      expect(value()).toBe("20.00");
+      expect(hiddenValue()).toBe("20.00");
```

Comment:
Do you need to assert that the component didn't error?

Reply:
My thinking behind this was that if anyone in the future makes any changes to the onBlur or the `formatValue`, we have tests to catch anything that will effect and deadkey characters. I'm happy to do this a different way if you know of a better way?

Reply:
Thats fine, but I'm not sure if this test would fail if the component did error, please can you check?

Usually we'd use `expect().not.toThrow()`

Reply:
No bother. I'll change the test to include `expect().not.toThrow()` as that is probably a better test for what I was wanting to achieve. Thanks for that 👍🏻